orbital4c/NuclearPotential.py

```diff
@@ -26,13 +27,6 @@ def uHFYGB(r):
     u = erf(r)/r + (1/(3*np.sqrt(np.pi)))*(np.exp(-(r**2)) + 16*np.exp(-4*r**2))
     return u
 
-def PoCh(position, center , charge):
-    d2 = ((position[0] - center[0])**2 +
-          (position[1] - center[1])**2 +
-          (position[2] - center[2])**2)
-    distance = np.sqrt(d2)
-    return charge / distance
-
 def HomChSph(position, center, charge, atom):
```

Comment:
Same here: 
e.g.  HomChSph --> PotentialHomogeneousSphere
and other places in the file with too short names :-)

Reply:
Fixed and pushed